src/server/launchpad/index.js

```diff
@@ -0,0 +1,14 @@
+/* Copyright 2016 Canonical Ltd.  This software is licensed under the
+ * GNU Affero General Public License version 3 (see the file LICENSE).
+ */
+
+import { conf } from '../helpers/config';
+import { Launchpad } from './client';
+
+export function getLaunchpad() {
+  return new Launchpad(
+    conf.get('LP_API_URL'), conf.get('LP_CONSUMER_KEY'),
```

Comment:
As it looks we need more env variables than just `LP_API_URL` that is added to dev env file we should have them documented at least.

I would suggest file similar to https://github.com/canonical-ols/javan-rhino/blob/master/docs/environment-variables.md (sorry we don't have it created yet - we should!).